Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/GradientTintedIconButton.kt

```diff
@@ -33,16 +41,55 @@ fun JetsnackGradientTintedIconButton(
     modifier: Modifier = Modifier,
     colors: List<Color> = JetsnackTheme.colors.interactiveSecondary
 ) {
+    val interactionSource = remember { MutableInteractionSource() }
+
     // This should use a layer + srcIn but needs investigation
+    val border = Modifier.fadeInDiagonalGradientBorder(
+        showBorder = true,
+        colors = JetsnackTheme.colors.interactiveSecondary,
+        shape = CircleShape
+    )
+    val pressed by interactionSource.collectIsPressedAsState()
+    val background =
+        if (pressed)
+            Modifier.offsetGradientBackground(
+                colors,
+                200f,
+                0f
+            )
+        else
```

Comment:
nit: please can we add braces on the if/else

Reply:
I have already added the braces